### ISSUE TITLE
Global variable query

### ISSUE THREAD

**Issue by MelMacaluso** [hodgef/js-library-boilerplate#589]
Hi, thanks for the boilerplate it saves lots of time.

Just a quick one, as far as I skim read to use the lib one's creates and if hosted on a CDN we need to first 

```
let MyLibrary = window.MyLibrary.default;
let libraryInstance = new MyLibrary();
```

is it not possible to just pollute the global scope straight away without calling the default prop?

and is it possible to rename the global variable? didn't find any options so far (as in a custom name rather than `MyLibrary`).

Thanks again 🙏

**Comment by hodgef** [comment]
Hey @MelMacaluso, check-out the Customization section in the readme to remove/change any placeholder stuff (especially step #3).

![image](https://user-images.githubusercontent.com/25509135/80762384-778ec300-8b0a-11ea-8365-060208fef92f.png)

In terms of what's deployed, currently we `export default` in `index.js`
![image](https://user-images.githubusercontent.com/25509135/80762590-d48a7900-8b0a-11ea-9c6a-799c6cbab62d.png)

So that's why we have to get it from `default`.

That being said, you can always change this file to something like this
![image](https://user-images.githubusercontent.com/25509135/80762701-0996cb80-8b0b-11ea-98ad-0862d766963f.png)

Basically not exporting a default module, but an object. Then you can instantiate your library like so:
```
const { myApp } = window.MyLibrary;
const libraryInstance = new myApp();
```

**Comment by MelMacaluso** [comment]
@hodgef 

Hi mate, thanks so much! I take the blame for not have read the customization part properly...and for not knowing how UMD handles the default exports ahaha.


Thanks again!
